acdc/acdc_graphics.py

```diff
@@ -190,6 +205,10 @@ def show(
                     g2.add_edge(s[i], s[j], style="invis", weight=200)
             g2.write(path=base_path / f"{k}.gv")
         g.write(path=base_fname + ".gv")
+
+        if not fname.endswith(".gv"):
```

Comment:
Hopefully https://github.com/ArthurConmy/Automatic-Circuit-Discovery/pull/79 can be merged